packages/create-hint/src/templates/meta.ts.hbs

```diff
@@ -1,21 +1,43 @@
 import { Category } from 'hint/dist/src/lib/enums/category';
 import { HintScope } from 'hint/dist/src/lib/enums/hint-scope';
 import { HintMetadata } from 'hint/dist/src/lib/types';
+{{#if packageData.official}}
+
+import { getMessage } from './i18n.import';{{/if}}
 
 const meta: HintMetadata = {
     docs: {
-        category: Category.{{category}},
-        description: `{{description}}`
+        category: Category.{{hint.category}},{{#if packageData.official}}
+        description: getMessage('{{#if packageData.isMulti}}{{hint.normalizedName}}_{{/if}}description', 'en'),
+        name: getMessage('{{#if packageData.isMulti}}{{hint.normalizedName}}_{{/if}}name', 'en'){{else}}
+        description: `{{hint.description}}`,
```

Comment:
any reason to use ` instead of '?

Reply:
Not any specific reason. We can remove this one.

Reply:
It was already there earlier, So I just left it as it is. But I think, it should not have any role here.

Reply:
I was just curious. The only thing is that we should escape the character used (I don't know if we are already doing that or not)

Reply:
Hmm right. Checking now if we are escaping or not. However backtick should be still safe to use in such cases.

Reply:
we are doing `escape` while creating `NewHint`
```
this.description = escapeSafeString(hintData.description);
```

Should I keep single-quote?

Reply:
nah, it is ok.